.github/workflows/publish-docs.yml

```diff
@@ -27,10 +27,13 @@ jobs:
           cache: pip
           cache-dependency-path: docs/requirements.txt
 
-      - name: Set up Go ${{ env.GO_VERSION }}
-        uses: actions/setup-go@v3
+      - name: Set up Go
+        uses: actions/setup-go@v5
```

Comment:
We could replace this step with a curl/wget if k0s_sort would be available as precompiled executable.